go/vt/tabletmanager/tmclient/rpc_client_api.go

```diff
@@ -83,8 +83,11 @@ type TabletManagerClient interface {
 	// ApplySchema will apply a schema change
 	ApplySchema(ctx context.Context, tablet *topo.TabletInfo, change *myproto.SchemaChange) (*myproto.SchemaChangeResult, error)
 
-	// ExecuteFetch executes a query remotely using the DBA pool
-	ExecuteFetch(ctx context.Context, tablet *topo.TabletInfo, query string, maxRows int, wantFields, disableBinlogs bool) (*mproto.QueryResult, error)
+	// ExecuteFetchAsDba executes a query remotely using the DBA pool
+	ExecuteFetchAsDba(ctx context.Context, tablet *topo.TabletInfo, query string, maxRows int, wantFields, disableBinlogs bool) (*mproto.QueryResult, error)
+
+	// ExecuteFetchAsApp executes a query remotely using the App pool
+	ExecuteFetchAsApp(ctx context.Context, tablet *topo.TabletInfo, query string, maxRows int, wantFields, disableBinlogs bool) (*mproto.QueryResult, error)
```

Comment:
I don't think we want to expose 'disableBinlogs' here, the app user probably doesn't even have the permissions to do it.